src/Mercure.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Bundle\MercureBundle;
+
+use Symfony\Component\Mercure\Exception\InvalidArgumentException;
+use Symfony\Component\Mercure\Hub;
+use Symfony\Component\Mercure\Jwt\TokenFactoryInterface;
+use Symfony\Component\Mercure\PublisherInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+final class Mercure
```

Comment:
Is this class really useful? IMHO it would be better to inject the hub directly (using autowiring alias) or a simple array/iterable of hubs if you need several hubs.